jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -230,6 +230,7 @@
                 export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
                 export KUBE_GCE_ZONE="us-central1-f" #TODO(colhom): This should be generalized out to plural case
                 export KUBE_GCS_RELEASE_BUCKET=kubernetes-federation-release
+                export KUBE_GCS_DEV_RELEASE_BUCKET_MIRROR=kubernetes-federation-release
```

Comment:
did you mean to add _MIRROR on this? in #28193 it was just `KUBE_GCS_DEV_RELEASE_BUCKET`.